etc/config/contextBroker

```diff
@@ -57,3 +57,10 @@ BROKER_DATABASE_NAME=orion
 # We need to start the CB with log append mode to not overwrite previous logs and logrotate work correctly
 BROKER_EXTRA_OPS="-multiservice -logAppend"
 
+# The next environment variable generates, if it is defined as 'true', an archive in the 
+# path /var/cb_cores with the core file and the logs when the process crash. When this var is 'true' 
```

Comment:
The directory has to exist? Or the core generation and archiving logic is able to create it? In the former case it should be mentioned in this piece of the documentation.

Reply:
When the variable is set to `true` all the logic, creating, archiving and making the rotation in archives is fully functional...

May be indicate that a `bzip2` utility should be installed because in the `tar` is used the option `j` to compress.

Reply:
Documentation adjusted in 6d0e9e9